### ISSUE TITLE
How to get the independent Probability

### ISSUE THREAD

**Issue by vikash512** [stratospark/food-101-keras#9]
How to get the independent Probability?
I am trying to detect the food images in composite image using CNN,In the final layer I have done the Softmax classifier to get the prediction,I am getting the result as sum of all probabilities equals to one.But I need to get the independent class probabilities. So can you guide me is there any function that can give the independent class probability.So can you guide me how to get the independent class Probability,Using which function.

Thank you.
